### ISSUE TITLE
azk without brew

### ISSUE THREAD

**Issue by kemelzaidan** [azukiapp/azk#473]
I try installing azk in Mac Yosemite but it failed:

```bash
 wget -qO- http://azk.io/install.sh | bash
[azk] Checking platform.......................................
[ DONE ]
[azk] Detected platform: darwin, x64
[azk] Checking for VirtualBox installation....................
[ DONE ]
[azk] Virtual Box detected
[ DONE ]
[azk] Installing azk..........................................
Warning: azk-0.14.2 already installed, it's just not linked
[ DONE ]

[azk] azk has been successfully installed.
krusty: kemelzaidan$ azk agent start
-bash: azk: command not found
```

I believe it's probably because I don't have brew installed on my system since I use MacPorts: https://www.macports.org. If that's right, brew is not listed as a requirement on the documentation.

I have nodejs installed on my system using [nvm](https://github.com/creationix/nvm) and it mustn't be hard to link to it if I know what brew link does and  what azk needs.

Thanks,

Kemel Zaidan

**Comment by fearenales** [comment]
Hi @kemelzaidan! Thanks for your feedback!

You're right, there's a hole in our docs: [homebrew](http://brew.sh/) is missing. We're going to fix it ASAP, but, for now, just install [homebrew](http://brew.sh/) and follow on our [installation instructions](http://docs.azk.io/en/installation/mac_os_x.html).

Any other issue, just let me know!

**Comment by fearenales** [comment]
@kemelzaidan looks like you already have `brew` installed (the message `Warning: azk-0.14.2 already installed, it's just not linked` indicates that). By our experience, having both `brew` and `macports` installed may cause problems. (http://superuser.com/questions/181337/is-it-safe-to-install-both-homebrew-and-macports-on-the-same-machine)

Since you already have `brew`, to install the latest azk version, just run:
```
$ brew update
$ brew install azukiapp/azk/azk
```

**Comment by kemelzaidan** [comment]
No, I don't have brew, only macports. The message says that because I had already previously tried to install it. In fact it did install but like on this try, it has failed on the link stage. I had to run it again for the only purpose of giving you some clue.

Thanks
